pyuvsim/tests/test_simsetup.py

```diff
@@ -235,7 +247,8 @@ def test_tele_parser():
     assert bdict == {}
 
     tdict.pop('array_layout')
-    simtest.assert_raises_message(KeyError, 'array_layout must be provided.', pyuvsim.simsetup.parse_telescope_params, tdict)
+    simtest.assert_raises_message(KeyError, 'array_layout must be provided.', pyuvsim.simsetup.parse_telescope_params,
+                                  tdict)
```

Comment:
A more general comment --- Are we now setting a maximum line length?

Reply:
My preference is to not have a hard cutoff but to strongly prefer line lengths of 80 or less unless longer lines help a lot with readability. I get that that is harder to stick to because there's no enforcement, but I think that unreasonably forcing line wrapping can substantially hurt readability in some cases. The way I do it is that I have a visual marker in my editor that shows me when I've gone beyond 80 characters and then I can choose to make the line shorter or not as a judgement call. But I'm open to another approach.

Reply:
I agree with @bhazelton . I think line length should be kept to if it makes sense to do so. I find that _almost always_ it's possible to do that. The one referenced here is an example where I think my editor auto-reformatted it. If I'd done it manually, I would have put all the arguments on a new line, with an indent. This is in conformance with `black`.

Reply:
OTOH, `flake8` has a strict line-limit (we can probably turn it off). I actually think ~88 columns is nicer. I guess if we really thnk a line is worth being really long, we can whack a `# noqa` on the end of it.

Reply:
Looks to me like we can ignore flake8 errors as we did for pycodestyle.

Reply:
In b7e89d70fab47c00273dbfa95fa5f5c82f3af8b0 I changed the `max-line-length` to 100, and un-ignored the line limit check. Is this what we want? I can just ignore the limit completely, and remove the redundant `max-line-length` if you prefer.

I do really think that having a line limit is a good idea when multiple people are contributing. We just have to agree on how the lines should be split.